packages/web/src/components/tipping/support/SupportingList.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useCallback } from 'react'
+
+import { IconArrow } from '@audius/stems'
+import { useDispatch } from 'react-redux'
+
+import { ReactComponent as IconTip } from 'assets/img/iconTip.svg'
+import { useSelector } from 'common/hooks/useSelector'
+import { getProfileUser } from 'common/store/pages/profile/selectors'
+import { getSupporting } from 'common/store/tipping/selectors'
+import {
+  setUsers,
+  setVisibility
+} from 'store/application/ui/userListModal/slice'
+import {
+  UserListEntityType,
+  UserListType
+} from 'store/application/ui/userListModal/types'
+
+import styles from './Support.module.css'
+import { SupportingTile } from './SupportingTile'
+
+const MAX_SUPPORTING_TILES = 3
+
+const messages = {
+  supporting: 'Supporting',
+  seeMorePrefix: 'See ',
+  seeMoreSuffix: ' More'
+}
+
+export const SupportingList = () => {
+  const dispatch = useDispatch()
+  const profile = useSelector(getProfileUser)
+  const supportingMap = useSelector(getSupporting)
+  const supportingForProfile = profile
+    ? supportingMap[profile.user_id] ?? {}
+    : {}
+  const rankedSupportingList = Object.keys(supportingForProfile)
+    .sort((k1, k2) => {
+      const id1 = parseInt(k1)
+      const id2 = parseInt(k2)
+      return supportingForProfile[id2].amount - supportingForProfile[id1].amount
+    })
+    .map(k => supportingForProfile[parseInt(k)])
```

Comment:
Any perf concerns here if we fetch a bunch of supporters for a profile by scrolling way down in the userlist, and then come back to their profile and have parseInt + sort to show the top 3?

Reply:
no longer applicable with new model